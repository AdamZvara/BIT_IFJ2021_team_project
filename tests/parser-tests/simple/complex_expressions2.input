require "ifj21"


function foo(n : number)
	local s : string = "ahoj"
	local num : number = 2.21 + 3.5
	write(num,"\n")
	num = 2.19 * n
	num = num - n
	if (((((1>0))))) then
		write(num,"\n")

	else end

end

foo(2.4e2)
