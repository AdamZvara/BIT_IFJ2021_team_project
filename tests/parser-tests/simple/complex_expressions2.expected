0x1.6d70a3d70a3d7p+2
0x1.1d9999999999ap+8
